caffe_app/views/DB.py

```diff
@@ -25,14 +25,26 @@ def save_to_db(request):
         net_name = request.POST.get('net_name')
         if net_name == '':
             net_name = 'Net'
-        try:
-            randomId = datetime.now().strftime('%Y%m%d%H%M%S')+randomword(5)
-            model = ModelExport(name=net_name, id=randomId, network=net, createdOn=datetime.now(),
-                                updatedOn=datetime.now())
+        if request.POST.get('modelID', 'null') != 'null':
+            model = ModelExport.objects.get(id=request.POST.get('modelID'))
+            model.network = net
+            model.updatedOn = datetime.now()
+            model.name = net_name
             model.save()
-            return JsonResponse({'result': 'success', 'id': randomId})
-        except:
-            return JsonResponse({'result': 'error', 'error': str(sys.exc_info()[1])})
+            return JsonResponse({'result': 'success', 'id': request.POST.get('modelID')})
+        else:
+            try:
+                randomId = datetime.now().strftime('%Y%m%d%H%M%S')+randomword(5)
```

Comment:
Can you please change `randomId` to `random_id` since we don't follow the camel casing in python.

Reply:
Also, can you please make sure that you follow the underscore styling everywhere in the code?